### PR TITLE
@broskoski => Match locations w/o accents, but keep displaying the accents

### PR DESCRIPTION
**Pull request by mzikherman**
Closes https://github.com/artsy/force/issues/67

### CODE REVIEW

File: components/commercial_filter/filters/location/location_filter_view.coffee [hunk 1]
```diff
@@ -26,12 +27,21 @@ module.exports = class LocationFilterView extends Backbone.View
   bloodHound: ->
     cities = _.pluck GEOCODED_CITIES, 'name'
     @hound = new Bloodhound
-      datumTokenizer: Bloodhound.tokenizers.whitespace,
-      queryTokenizer: Bloodhound.tokenizers.whitespace,
-      local: cities
+      datumTokenizer: Bloodhound.tokenizers.obj.whitespace('value'),
+      queryTokenizer: @queryTokenizer,
+      local: _.map(cities, (city) => { value: @translate(city), display: city })
```
Comment (mzikherman) [inline components/commercial_filter/filters/location/location_filter_view.coffee]: return the `value` which gets used in the tokenizers and has the accented characters removed, but also return the unmodified `display` to render.

**Comment by mzikherman** [comment]
bp

**Comment by broskoski** [comment]
ṿëȑẙ ñïćē

**Comment by mzikherman** [comment]
At first I switched order of the diacritics, so all the cities were coming thru like `ñéw Yôrk, ñY, ûSà`.

**Pull request merged**